spec/support/fake_heroku.rb

```diff
@@ -6,8 +6,9 @@ def initialize(args)
   end
 
   def run!
-    if @args.first == "plugins"
-      puts "heroku-pipelines@0.29.0"
+    if @args.first == "help"
+      # spaces in front are intentional
+      puts "  pipelines      #  manage collections of apps in pipelines"
```

Comment:
I mean does it work, and does it provide the same level of confidence as the one with spaces. Because if it does, I'd make spaces optional, which would remove the need to have the comment.